src/shogun/neuralnets/RBM.cpp

```diff
@@ -84,7 +84,7 @@ void CRBM::add_visible_group(int32_t num_units, ERBMVisibleUnitType unit_type)
 			m_visible_state_offsets->element(n-1)+m_visible_group_sizes->element(n-1));
 }
 
-void CRBM::initialize(float64_t sigma)
```

Comment:
Lets name it ```initialize_neural_net``` as well. It does initialize weights now but if we change something we'd have to rename it.